_posts/2024-11-08-socratic-seminar-30.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+type: socratic
+title: "Seminário Socrático 030"
+meetup: https://www.meetup.com/clubebitcoinsp/events/303963950
+---
+
+## Avisos
+
+- Entrem no grupo do Whatsapp [São Paulo Bitdevs](https://chat.whatsapp.com/HiaPqjmUqER5djFPR1Yl3T)!
+- Respeite a privacidade dos participantes.
+- Os meetups nunca são gravados. Queremos todos a vontade para participar e discutir os assuntos programados, de forma anônima se assim o desejarem.
+
+## Agradecimentos
+
+- Agradecemos à Vinteum pela casa, comidas e bebidas.
+
+## Cronograma
+
+### Warmup
+
+* [Building Bitcoin - Funding a $1.2 Trillion Dollar Project](http://1a1z.com/fund.html)
+* [Send Bitcoin to anyone that has an email address. (It uses MPC)](https://emailbtc.net/)
+
+## Bitcoin
+
+* [Disclosure of CVE-2024-35202](https://bitcoincore.org/en/2024/10/08/disclose-blocktxn-crash/)
+* [Disclosure of CVE-2024-35202](https://bitcoincore.org/en/2024/10/08/disclose-blocktxn-crash/)
+* [CVE-2024-38365 public disclosure (btcd `FindAndDelete` bug)](https://delvingbitcoin.org/t/cve-2024-38365-public-disclosure-btcd-findanddelete-bug/1184?u=antoinep)
+* [Disclosure of DoS due to inv-to-send sets growing too large](https://bitcoincore.org/en/2024/10/08/disclose-large-inv-to-send)
+* [Disclosure of hindered block propagation due to mutated blocks](https://bitcoincore.org/en/2024/10/08/disclose-mutated-blocks-hindering-propagation)
+* [Guide for Wallets Employing Bitcoin Core 28.0 Policies](https://bitcoinops.org/en/bitcoin-core-28-wallet-integration-guide/)
+* [Deprecated RPC on Bitcoin Core v28.0 prevents LND from starting](https://x.com/roasbeef/status/1842956355516223824)
+* [Bitcoin Core v28.0 released](https://github.com/bitcoin/bitcoin/releases/tag/v28.)
+
+### Bitcoin Q&A
+
+* [Bitcoin Core 28 Tests (testmempoolaccept rejected but submitpackage accepted)](https://bitcoin.stackexchange.com/questions/124269/bitcoin-core-28-tests-testmempoolaccept-rejected-but-submitpackage-accepted)
+* [How does ban score algorithm calculate ban score for a node?](https://bitcoin.stackexchange.com/questions/117227/how-does-ban-score-algorithm-calculate-ban-score-for-a-node)
+* [What specific verifications are done on a fresh Bitcoin TX and in what order?](https://bitcoin.stackexchange.com/questions/124221/what-specific-verifications-are-done-on-a-fresh-bitcoin-tx-and-in-what-order)
+
+### Lightning
+
+* [LN Summit 2024 summary](https://delvingbitcoin.org/t/ln-summit-2024-notes-summary-commentary/1198)
+* [Phoenix moved its LSP to a new open protocol with an open-source implementation](https://x.com/PhoenixWallet/status/1844377194489053555)
+
+### Cryptography
+
+* [MuSig2 merged into secp256k1 repo](https://github.com/bitcoin-core/secp256k1/pull/1479)
+
+### Mining
+
+* [OCEAN - DATUM](https://ocean.xyz/docs/datum)
+* [MDK Beta Update: Early Feedback & Expanded Participation](https://www.mining.build/blog/mdk-beta-update/)
+    - [www.mining.build](https://www.mining.build/manuals/)
+
```

Comment:
```suggestion

### Layer 2 ++

* [Alpen Labs launches vanilla bitcoin signet with a BitVM-based bridge and Zero-knowledge commitments on-chain](https://www.stratabtc.org)
```

Can I do a shameless plug?

Reply:
added that for you

Reply:
[adding storopoli shameless plug](https://github.com/saopaulobitdevs/saopaulobitdevs.org/pull/39/commits/ee85b036ffe7dbf364b688f231b6dbe97548a591)